src/components/Logo/QueryTokenLogo.tsx

```diff
@@ -1,32 +1,18 @@
-import { NATIVE_CHAIN_ID } from 'constants/tokens'
-import { TokenStandard } from 'graphql/data/__generated__/types-and-hooks'
+import { PortfolioLogo } from 'components/AccountDrawer/MiniPortfolio/PortfolioLogo'
 import { SearchToken } from 'graphql/data/SearchTokens'
 import { TokenQueryData } from 'graphql/data/Token'
 import { TopToken } from 'graphql/data/TopTokens'
-import { supportedChainIdFromGQLChain } from 'graphql/data/util'
+import { gqlToCurrency, supportedChainIdFromGQLChain } from 'graphql/data/util'
 
-import AssetLogo, { AssetLogoBaseProps } from './AssetLogo'
+import { AssetLogoBaseProps } from './AssetLogo'
 
 export default function QueryTokenLogo(
   props: AssetLogoBaseProps & {
     token?: TopToken | TokenQueryData | SearchToken
   }
 ) {
-  const chainId = props.token?.chain ? supportedChainIdFromGQLChain(props.token?.chain) : undefined
+  const chainId = (props.token?.chain ? supportedChainIdFromGQLChain(props.token?.chain) : 1) ?? 1
+  const currency = props.token ? gqlToCurrency(props.token) : undefined
 
-  return (
-    <AssetLogo
-      isNative={
-        // TODO(cartcrom): simplify this check after backend fixes token standard on assetActivities tokens
-        !props.token?.address ||
-        props.token?.standard === TokenStandard.Native ||
-        props.token?.address === NATIVE_CHAIN_ID
-      }
-      chainId={chainId}
-      address={props.token?.address}
-      symbol={props.token?.symbol}
-      backupImg={props.token?.project?.logoUrl}
-      {...props}
-    />
-  )
+  return <PortfolioLogo currencies={[currency]} chainId={chainId} {...props} />
```

Comment:
ooh this `[currency]` should be memoized to prevent rerendering

Reply:
gr8 point, i think this is an issue elsewhere, I made a ticket to fix this across all PortfolioLogo callsites